.gitignore

```diff
@@ -0,0 +1,5 @@
+__pycache__/
+*.egg-info/
+*.pyc
+.coverage
+
```

Comment:
Let's omit this file. See https://blog.jaraco.com/skeleton/#ignoring-artifacts

Reply:
Removed from the repo